### DISCUSSION TITLE
Conversion from computer case to human readable

### DISCUSSION THREAD

**Discussion by aure** [krzysztofzablocki/SourceryPro-Feedback#6] [Ideas]
I think in Ruby its called "humanize" but what I'd like is to convert things as follows:

isDebugModer => "Is Debug Mode"
dataModel => "Data Model"

Is this already doable? I don't think it is because it could be immediately useful in say the SwiftUI Form template.

**Comment by krzysztofzablocki** [comment]
I don't think that filter exists atm, I'll add it to the list to implement in main Sourcery
